mesa/space.py

```diff
@@ -349,6 +366,7 @@ def _place_agent(self, pos: Coordinate, agent: Agent) -> None:
     def remove_agent(self, agent: Agent) -> None:
         """ Remove the agent from the grid and set its pos variable to None. """
         pos = agent.pos
+        assert isinstance(pos, Tuple[int, int])
```

Comment:
Why is it hardcoded to be `Tuple[int, int]`, not `Coordinate`?

Reply:
Because otherwise mypy throws an error of "The type alias to Tuple is invalid in runtime context". However, at runtime `Tuple[int, int]` throws an "TypeError: Subscripted generics cannot be used with class and instance checks", so neither actually works. I will see what to do about this.